src/main/resources/org/openstreetmap/atlas/checks/validation/tag/invalid-tags-check-regex-filter.json

```diff
@@ -318,6 +318,60 @@
         "with"
       ],
       "exceptions": []
-    }    
+    },
+    {
+      "instruction": "The tag: {0} contains badly written word etang, please verify if accent and capital first letter \u00C9tang should be used instead.",
```

Comment:
I'm not sure we should be adding rules for specific typos here. If we go down that rabbit hole we could end up with a massive config. I would suggest creating a new check for common typos, that employs a more compact word definition.

Reply:
I added it here because of several considerations:
1) The number of typo reported in OSMOSE is not that many. It's the same words that are repeated in the French speaking countries
2) If it were separated into a separate test, there is a lot of code copying with similar logic and sonarcloud code smell will complain
3) The test run will be more efficient this way because we don't have to run the tags twice for 2 separate tests
4) The current regex test is just too good...

This is the last word tag check on my list for this year. If there are similar tests in the future, I will certainly separate them into new tests.